src/DataViews/include/DataViews/DataView.h

```diff
@@ -175,8 +175,8 @@ class DataView {
   void OnStopDownloadRequested(const std::vector<int>& selection);
   virtual void OnSelectRequested(const std::vector<int>& selection);
   virtual void OnUnselectRequested(const std::vector<int>& selection);
-  void OnEnableFrameTrackRequested(const std::vector<int>& selection);
-  void OnDisableFrameTrackRequested(const std::vector<int>& selection);
+  virtual void OnEnableFrameTrackRequested(const std::vector<int>& /*selection*/) {}
```

Comment:
Since we now have a subclass, why is this method still on this interface here? Wouldn't it make sense to only have this on ScopeDataViews?

Reply:
As said. That feels strange. But should not be a problem of this PR. Thanks for fixing the actual issue.

Reply:
Yes. Ideally, that would be composable (I am missing traits and mixins ....). Now that is kind of hidden by this action status. So we try to add all actions to the DataView class, and if the action is not available on the actual instance, we do not add it. But we need to handle everything on the super-class.

Anyways, nothing that should keep you from merging your PR.

Reply:
> Yes. Ideally, that would be composable (I am missing traits and mixins ....)
can't this be done via multiple/virtual inheritance?